.claude/scripts/mcp_common.sh

```
@@ -0,0 +1,2130 @@
+        log_with_timestamp "Server github-server already exists, skipping"
+        INSTALL_RESULTS["github-server"]="ALREADY_EXISTS"
+        SUCCESSFUL_INSTALLS=$((SUCCESSFUL_INSTALLS + 1))
+    else
+        echo -e "${BLUE}  🔗 Adding GitHub official remote MCP server...${NC}"
+        log_with_timestamp "Adding GitHub official remote MCP server"
+
+        # Remove any old deprecated GitHub server first
+        safe_remove "github-server"
+
+        # Add the new official GitHub HTTP MCP server (secure: token via temporary file)
+        local temp_config=$(mktemp -t github_mcp.XXXXXX)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)